browser/help/topics/regional-constraint.md

```diff
@@ -3,40 +3,37 @@ id: regional-constraint
 title: 'Regional constraint'
 ---
 
-Important: Currently regional constraint is only available / displayed when selecting the ExAC dataset. In addition, it is only displayed for genes that exhibit regional missense constraint.
+**Important**: The regional missense constraint track is currently only available / displayed when selecting the gnomAD v2 dataset. In addition, there are three distinct views:
+- Regional constraint information for genes that exhibit evidence of regional missense constraint (RMC)
+- Transcript-wide missense information for genes that were searched for but did not exhibit any evidence of RMC
+- Text for [outlier genes](https://gnomad.broadinstitute.org/help/why-are-constraint-metrics-missing-for-this-gene-or-annotated-with-a-note) not searched for evidence of RMC
 
 ### Overall interpretation
 
-We searched for regions within transcripts that were intolerant of missense variation within the ExAC dataset. We used the observed and expected missense variation in each transcript in a likelihood ratio test to identify those transcipts that had two or more regions with significantly different levels of missense constraint (as measured by depletion of expected missense variation). Missense constraint values closer to zero indicate increased intolerance against missense variation.
+We searched for regions within transcripts that were intolerant of missense variation within the gnomAD v2.1.1 dataset. We used the observed and expected missense variation in each transcript in a likelihood ratio test to identify those transcripts that had two or more regions with significantly different levels of missense constraint (as measured by depletion of expected missense variation). Missense constraint values closer to zero indicate increased intolerance against missense variation.
 
-Note that these data currently reflect the regional constraint seen in the ExAC dataset. They will be updated to gnomAD in the near future.
+Note that these data currently reflect the regional constraint seen in gnomAD v2.1.1, which was mapped to GRCh37. We will update to GRCh38 in the near future.
 
 More details can be found in [Samocha et al bioRxiv 2017](https://www.biorxiv.org/content/early/2017/06/12/148353).
```

Comment:
We could also note that expected counts were determined as in the 2020 paper.

I also wonder about the preprint since our approach has substantially changed. Potentially it could be "previous work" and a note about an upcoming blog post to explain changes to the algorithm?

Reply:
Ah, I see we link to the 2020 paper below, so ignore that comment.